### DISCUSSION TITLE
In App purchase

### DISCUSSION THREAD

**Discussion by santoshgontu** [elementary/appcenter#1510] [Ideas]
Like other App store platforms. Introducing a in App purchase to App centre apps will be great.

**Comment by MvBonin** [comment]
I think, the whole point of the Appcenter is, that the user can pay what he wants to support the developers. In this way everyone has access to the software and all the functionality of the software. 
By adding in-app purchases, the user would have to pay for functionality inside the apps. Apps would have paid pro versions. That would destroy the whole "pay-what-you-want" (and can) and "everyone can have access to good software" system.
For me it is very refreshing not to have in-app purchases (or ads) and its one of the reasons I like the philosophy of AppCenter.
